eldritch/encounters.py

```diff
@@ -129,22 +129,137 @@ def Lodge1(char):
   check = events.Check(char, "lore", -1)
   draw = events.Draw(char, "spells", 2)
   return events.PassFail(char, check, draw, events.Nothing())
+def Lodge2(char):
+  check = events.Check(char, "fight", -1)
+  ally = events.DrawSpecific(char, "allies", "Thief") # TODO: prereq on the ally being in the deck
+  return events.PassFail(char, check, ally, events.Nothing())
+def Lodge3(char):
+  check = events.Check(char, "luck", -1)
+  curse = events.BlessCurse(char, False)
+  return events.PassFail(char, check, events.Nothing(), curse)
+def Lodge4(char):
+  prereq = events.AttributePrerequisite(char, "dollars", 3, "at least")
+  pay = events.Loss(char, {"dollars": 3})
+  check = events.Check(char, "will", -1)
+  damage = events.Loss(char, {"stamina": 2})
+  membership = events.MembershipChange(char, True)
+  resist = events.PassFail(char, check, events.Nothing(), damage)
+  move = events.ForceMovement(char, "FrenchHill")
+  choice = events.BinaryChoice(char, "Join the Lodge?", "Yes", "No",
+                               events.Sequence([pay, membership], char), resist)
+  return events.Sequence([events.PassFail(char, prereq, choice, resist), move], char)
```

Comment:
The way I read this event, you move to the street regardless of whether you take the stamina damage or not, but you don't move to the street if you join the order.